core/src/main/java/de/danoeh/antennapod/core/util/FeedUtils.java

```diff
@@ -0,0 +1,69 @@
+package de.danoeh.antennapod.core.util;
+
+import android.database.Cursor;
+
+import androidx.annotation.NonNull;
+
+import de.danoeh.antennapod.core.feed.Feed;
+import de.danoeh.antennapod.core.feed.FeedPreferences;
+import de.danoeh.antennapod.core.storage.PodDBAdapter;
+
+/**
+ * Helper methods for using the {@link Feed} class.
+ */
+public abstract class FeedUtils {
```

Comment:
I don't like generic "Utils" classes. We already have a ton of them and you never know which util class to use. I think it would be better to move it to the `storage` package and call it something like `FeedCursorMapper` (see also [CursorMapper](https://developer.android.com/reference/androidx/leanback/database/CursorMapper). I don't think we need to actually extend that class but we could use the same interface)

Reply:
Nice idea, I updated the code accordingly.